### PR TITLE
fix: refactor functions and code

### PR DESCRIPTION
**Pull request by leo220yuyaodog**

### CODE REVIEW

File: web/src/common/SamlWidget.js [hunk 1]
```diff
@@ -1,3 +1,17 @@
+// Copyright 2023 The Casdoor Authors. All Rights Reserved.
```
Comment (hsluoyz) [inline web/src/common/SamlWidget.js:1]: Is it created in 2023?

File: util/string.go [hunk 2]
```diff
@@ -227,7 +227,7 @@ func IsChinese(str string) bool {
 }
 
 func GetMaskedPhone(phone string) string {
-	return getMaskedPhone(phone)
+	return rePhone.ReplaceAllString(phone, "$1****$2")
```
Comment (hsluoyz) [inline util/string.go:230]: Why not remove getMaskedPhone?

File: object/check_util.go [hunk 3]
```diff
@@ -22,6 +22,11 @@ import (
 	"github.com/casdoor/casdoor/i18n"
 )
 
+const (
+	LoginWrongTimesLimit      = 5
+	LastSignWrongTimeDuration = time.Minute * 15
```
Comment (hsluoyz) [inline object/check_util.go:27]: LastSignWrongTimeDuration -> LastSigninWrongTimeDuration

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: controllers/service.go [hunk 4]
```diff
@@ -130,13 +130,13 @@ func (c *ApiController) SendSms() {
...
 
-	org := object.GetOrganization(smsForm.OrgId)
 	var invalidReceivers []string
 	for idx, receiver := range smsForm.Receivers {
+		// The receiver phone format: E164 like +8613854673829 +441932567890
 		if !util.IsPhoneCnValid(receiver) {
 			invalidReceivers = append(invalidReceivers, receiver)
 		} else {
-			smsForm.Receivers[idx] = fmt.Sprintf("+%s%s", org.PhonePrefix, receiver)
+			smsForm.Receivers[idx] = receiver
```
Comment (hsluoyz) [inline controllers/service.go:139]: Does your change work?

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@leo220yuyaodog fix:

image

File: util/string.go (hunk 2 above)
Comment (leo220yuyaodog) [reply util/string.go]: `getMaskedPhone` no usage found in other places

File: controllers/service.go (hunk 4 above)
Comment (leo220yuyaodog) [reply controllers/service.go]: It's work

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**